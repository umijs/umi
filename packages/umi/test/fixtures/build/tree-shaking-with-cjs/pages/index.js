import foo from '../foo';

export default function() {
  return (
    <div>
      <h1>Index Page {foo}</h1>
    </div>
  );
}
