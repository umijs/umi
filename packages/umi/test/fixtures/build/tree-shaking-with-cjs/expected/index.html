<!DOCTYPE html><html><head>

<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">

<script>
  window.routerBase = "/";
</script>
</head>
<body>

<div id="root"></div>

<script src="/umi.js"></script>


</body></html>